src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Trim.cs

```diff
@@ -38,13 +38,13 @@ public TrimEndsFunction()
 
         public override DelegationCapability FunctionDelegationCapability => DelegationCapability.Trim;
 
-        public override bool IsRowScopedServerDelegatable(CallNode callNode, TexlBinding binding, OperationCapabilityMetadata metadata)
+        public override bool IsRowScopedServerDelegatable(CallNode callNode, TexlBinding binding, OperationCapabilityMetadata metadata, bool nodeInheritsRowScope = false)
```

Comment:
>IsRowScopedServerDelegatable [](http://example.com/codeflow?start=29&length=28)

This was already there, but do we need this override at all? If it only calls the base implementation, we can remove that.